### ISSUE TITLE
cve-2021-36260 ？

### ISSUE THREAD

**Issue by yyheart** [oldteamhost/nesca4#2]
Adding this cve is the most powerful one

**Comment by yyheart** [comment]
36260 can obtain the administrator's password instead of brute force cracking

**Comment by Lomasterrrr** [comment]
Yes, I looked at the code examples of this CVE, but it is used to hack hikvision web cameras. And I'm hacking hikvision IP cameras. But in principle it is pretty simple, and you can try it, but again, I still need to get the basics to a better level. And then I will add CVE.

**Comment by yyheart** [comment]
fighting

**Comment by yyheart** [comment]
Now?

**Comment by shrpow** [comment]
> Now?

he is died
